tensorrt_llm/serve/cluster_storage.py

```
@@ -187,26 +189,47 @@ def __init__(self,
         self._lock = asyncio.Lock()
         self._watch_handles = {}
         self._watch_lock = asyncio.Lock()
         self._check_expired_task = None
         self._check_expired_interval = 1  # in seconds
+        self._api_key = api_key
         if server:
             self.add_routes(server)
 
+    def _verify_api_key(
+            self,
+            credentials: HTTPAuthorizationCredentials = Security(HTTPBearer()),
```

Ruff (0.15.9)

[warning] Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)